js/web/lib/wasm/jsep/webgpu/ops/split.ts

```diff
@@ -25,6 +25,8 @@ const createSplitAttributesFromInputs =
       const splitSizes: number[] = [];
       if (inputs[1].dims[0] > 0) {
         inputs[1].getBigInt64Array().forEach(v => splitSizes.push(Number(v)));
+        attributes.splitSizes = splitSizes;
```

Comment:
Is it necessary to change input parameter, attributes. This function is supposed to return new/updated attributes.

Reply:
ah, yes can fix that.